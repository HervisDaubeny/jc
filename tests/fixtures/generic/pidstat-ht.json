[{"time":1692579199,"uid":0,"pid":1,"percent_usr":0.0,"percent_system":0.07,"percent_guest":0.0,"percent_wait":0.08,"percent_cpu":0.08,"cpu":0,"command":"systemd"},{"time":1692579199,"uid":0,"pid":1,"usr_ms":2890,"system_ms":4260,"guest_ms":0,"command":"systemd"}]
