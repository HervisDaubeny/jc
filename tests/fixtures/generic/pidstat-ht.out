Linux 4.18.0-477.10.1.el8_8.x86_64 (localhost.localdomain) 	08/20/2023 	_x86_64_	(1 CPU)

# Time        UID       PID    %usr %system  %guest   %wait    %CPU   CPU  Command
1692579199      0         1    0.00    0.07    0.00    0.08    0.08     0  systemd

# Time        UID       PID    usr-ms system-ms  guest-ms  Command
1692579199      0         1      2890      4260         0  systemd

